### PR TITLE
Benefits Blog Post

### LINKED ISSUE #4: Add a benefits section please.
It would go along way to add a benefits / why this is awesome section in the readme.  

Some questions might be: 
- what value does this provide?  
- why not just use public cran? 
- In which situation is this cran setup better than the regular public cran push-pull?

A flashy example would add value too.

### PR DESCRIPTION
**Pull request by ngparas**
Added a post about why the project is important, fixes #4

### CODE REVIEW

File: docs/why_cran_server.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+[As our colleague, Stephanie Kirmer,  has spoken about](https://github.com/skirmer/odsc2018/), embracing a practice of internal R package development on your team facilitates reproducibility, improves productivity and helps break down information silos. R-users who are building models and performing analyses are able to be more productive than they would be on their own because they can rely on code that’s already tested and documented rather than repeating common tasks. By capturing common processes, documentation and best-practices into knowledge-transfer artifacts, R packages speed up the onboarding process for new analytics professionals and empower other members of the organization.
```
Comment (jameslamb) [inline docs/why_cran_server.md]: cc @skirmer you're famous

File: docs/why_cran_server.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+#### Use a paid service like [Rstudio Package Manager](https://www.rstudio.com/products/package-manager/) 
+Rstudio Package Manager is new and appears to have a few very promising features. Though, it is a young, paid, closed-source solution. `cran-server` has been running internally at Uptake for nearly 2 years, serving thousands of downloads and over 60 packages. And, it’s free, lightweight and [open source](https://github.com/UptakeOpenSource/cran-server)!
+
+#### Use an Artifact repository like [Nexus](https://www.sonatype.com/nexus-repository-sonatype) or [Artifactory](https://jfrog.com/artifactory/)
```
Comment (ntdef) [inline docs/why_cran_server.md]: Kind of weird to write about a non-solution here. If you were just reading the headers, you'd be confused.

Comment (jameslamb) [reply docs/why_cran_server.md]: this one deserves a note but I agree with Troy that it isn't a solution and feels out of place. Maybe change the header to "write you own integration with an Artifact repository"?

Comment (ngparas) [reply docs/why_cran_server.md]: updated the header, does that fix the concern?

File: docs/why_cran_server.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Why You Should Run Your Own CRAN Server
```
[the remark this answers is not in the document]
Comment (ngparas) [reply docs/why_cran_server.md]: good idea, added

**Comment by ntdef** [comment]
I like this. I know we still want to add a gif. But we can open a new PR for that. 

Thanks again @ngparas!

**Pull request merged**